charts/incubator/spotweb/values.yaml

```
@@ -0,0 +1,56 @@
+image:
+  repository: jbaten/spotweb
+  pullPolicy: IfNotPresent
+  tag: 1.5.1@sha256:1008eeb207c326f5e1af86058d9591771fa9b111c188544467d42c4862b60fd7
```

```suggestion
image:
  repository: tccr.io/truecharts/spotweb
  pullPolicy: IfNotPresent
  tag: v1.5.1
```
Update with digest once https://github.com/truecharts/containers/pull/77 is merged